esprima.js

```diff
@@ -11,6 +11,11 @@
 var esprima = require('esprima-fb');
 var estraverse = require('estraverse-fb');
 var escodegen = require('escodegen');
+var listver = require('listver');
+
+var webpackMajorVersion = parseInt(
```

Comment:
https://github.com/webpack/webpack/blob/master/lib/NormalModule.js#L86 

Does for the loader api as seen above from `loaderContext.version`. However I'll have to look into where we are storing the new version info in webpack 2.